pkg/mover/chart.go

```diff
@@ -236,17 +285,116 @@ func namespacedPath(fullpath, chartName string) string {
 }
 
 /*
-Move executes the Chart relocation which includes
+  Move performs the relocation.
+
+A regular move executes the Chart relocation which includes
 
 1 - Push all the images to their new locations
 
 2 - Rewrite the Helm Chart and its subcharts
 
 3 - Repackage the Helm chart as toChartFilename
+
+A save to an offline tarball bundle will:
+
+1 - Drop all images to disk, with the original chart (unpacked) and hints file
+
+2 - Package all in a single compressed tarball
 */
 func (cm *ChartMover) Move() error {
+	if cm.targetOfflineTar != "" {
+		return cm.saveOfflineBundle()
+	}
+	return cm.moveChart()
+}
+
+func (cm *ChartMover) saveOfflineBundle() error {
 	log := cm.logger
 
+	tarPath, err := os.MkdirTemp("", "offline-tarball-*")
+	if err != nil {
+		return fmt.Errorf("failed to create temporary directory to build tar: %w", err)
+	}
+	log.Printf("Writing chart at %s/...\n", cm.chart.Metadata.Name)
```

Comment:
Would you mind adding empty lines between sections?

Reply:
Done